src/Directory.Build.props

```diff
@@ -71,6 +71,25 @@
     <DefineConstants>$(DefineConstants);ENCLAVE_SIMULATOR</DefineConstants>
   </PropertyGroup>
 
+  <!-- Packaging -->
+  <PropertyGroup>
+    <DebugType>portable</DebugType>
+    <DebugSymbols>true</DebugSymbols>
+    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
+    <IncludeSymbols>true</IncludeSymbols>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <EmbedUnTrackedSources>true</EmbedUnTrackedSources>
+  </PropertyGroup>
+  <!-- Packaging Extension CI Builds -->
+  <PropertyGroup Condition="'$(TF_BUILD)' == 'true'">
+    <!-- Repository Url is required for none SDK project even if it is already in nuspec file. -->
```

Comment:
If this is only for netfx project, could it be included in its csproj file?

Reply:
This block with TF_BUILD was deleted as the NugetPackProperties in build.proj is the actual properties used.